back/engines/commercial/multi_tenancy/db/seeds/community_monitor.rb

```diff
@@ -5,23 +5,34 @@
 module MultiTenancy
   module Seeds
     class CommunityMonitor < Base
+      def initialize(runner:, num_quarters: 2, ai_responses: false, locale: nil)
+        @num_quarters = num_quarters
+        @ai_responses = ai_responses
+        @locale = locale ||
+                  Locale.monolingual&.to_s ||
+                  AppConfiguration.instance.settings('core', 'locales').first ||
+                  I18n.default_locale
```

Comment:
The part `Locale.monolingual&.to_s || AppConfiguration.instance.settings('core', 'locales').first || I18n.default_locale` could be replaced by `Locale.default`. I would also suggest passing a Locale instance rather than a string for robustness (this is why the class was introduced, but it would be high effort to replace all occurrences).

Do we have tenants without any locale? If not, then `|| I18n.default_locale` can be removed. If yes, then I think we could add this to `Locale.default`.